tools/generator/test/Target+Testing.swift

```diff
@@ -28,7 +28,7 @@ extension Target {
     ) -> Self {
         return Target(
             name: product.name,
-            label: label ?? "//some/package:\(product.name)",
+            label: label ?? .init("//some/package:\(product.name)")!,
```

Comment:
Shouldn't this be covered by the string literal conformance?

Reply:
Unfortunately, no. It is a `String`, not a `StringLiteral`. The substitution keeps it from being a `StringLiteral`.